src/util/Injected.js

```diff
@@ -13,7 +13,11 @@ exports.ExposeStore = (moduleRaidStr) => {
     window.Store.Cmd = window.mR.findModule('Cmd')[0].Cmd;
     window.Store.CryptoLib = window.mR.findModule('decryptE2EMedia')[0];
     window.Store.DownloadManager = window.mR.findModule('downloadManager')[0].downloadManager;
-    window.Store.MDBackend = window.mR.findModule('isMDBackend')[0].isMDBackend();
+    if (window.mR.findModule('isMDBackend')[0]?.isMDBackend) {
```

Comment:
Missed this - we can't use `?.` since we still support Node v12, which doesn't have this.

Reply:
Got a problem here.